### PR TITLE
Add ULS language selector & create global static folder

### PR DESCRIPTION
**Pull request by cydrobolt**
- Add `static/global`, builds to `/static/`
- Add ULS language selector:
  - Remove old language selector
  - Add needed files to global static folder
  - Set up build script to work with ULS

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -45,16 +48,7 @@ <h3 class="masthead-brand">${title}</h3>
                 % endfor
 
                 <li class="dropdown">
```
Comment (ralphbean) [inline templates/index.html]: This whole dropdown entry can probably be removed from the top navbar now that the uls language selector is in the footer, no?

Comment (cydrobolt) [inline templates/index.html]: Oops. You're right

**Comment by cydrobolt** [comment]
Does the freeze affect this?

**Comment by ralphbean** [comment]
> Does the freeze affect this?

I don't think so.  It (1) isn't hosted in infra and (2) doesn't play a role in preparing the alpha release, so I think we're free to make changes.

**Comment by nirik** [comment]
Yeah, no freeze concerns here... should be able to make whatever changes you need. ;)

**Comment by cydrobolt** [comment]
@ralphbean I took away the unneeded dropdown. What do you think?

**Comment by ralphbean** [comment]
@Cydrobolt I had to make only one little change in 10938ef, but it looks good.  Thanks for this!

It's live up on http://whatcanidoforfedora.org now :rocket:

**Comment by cydrobolt** [comment]
On http://whatcanidoforfedora.org, the global static files aren't in the correct place. Works locally, though. Maybe something with OpenShift? @ralphbean

**Comment by ralphbean** [comment]
> the global static files aren't in the correct place

Really?
- How can you tell?
- Where should they be?

Did you try doing a hard-reload of the page?

**Comment by cydrobolt** [comment]
Yes, tried hard-reload. I know the global static files aren't correct because the ULS selector is missing images. Look in console

**Comment by ralphbean** [comment]
A data point:  it is looking for svgs, but the pngs are the only things there:  http://whatcanidoforfedora.org/static/img/

**Comment by cydrobolt** [comment]
Hmm. Shouldn't the SVGs be in there too?

**Comment by cydrobolt** [comment]
Yeah, you're right. It looks like I'll have to add the SVGs back in.

**Comment by ralphbean** [comment]
> Hmm. Shouldn't the SVGs be in there too?

Ah ha!  You never committed them.  They are ignored in the `.gitignore` file.  Can you add them explicitly and push to the uls-selector branch?

**Comment by cydrobolt** [comment]
Mhm. Is there a reason why the SVGs are in the `.gitignore`?

**Comment by cydrobolt** [comment]
Added in 1f6615e7228238b592baef13d82342fc12acd21b

**Comment by ralphbean** [comment]
> Mhm. Is there a reason why the SVGs are in the .gitignore?

To ignore the graphs produced by running `asknot-ng.py --graph`.

**Comment by ralphbean** [comment]
It looks better now, but http://whatcanidoforfedora.org/static/img/world_map.svg is still missing.

**Comment by cydrobolt** [comment]
Added it in 6b9ad8d705e8f65d18990abed87962a37efdc0d0
Apparently, it wasn't in the ULS source, so I had to dig it up from gf.o

**Comment by ralphbean** [comment]
Looks good!  Thanks @Cydrobolt!

**Pull request merged**